### ISSUE TITLE
loc loss became inf

### ISSUE THREAD

**Issue by jjjkkkjjj** [jjjkkkjjj/pytorch_TextBoxesPP#1]
When `ICDARFocusedSceneTextDataset` and `ICDARBornDigitalTextDataset`' was trained, loc loss was inf.
The reason is that "localization height or width = 0"

**Comment by jjjkkkjjj** [comment]
The reason was "incomprehensible" ground truth such like;

`gt_img_312.txt` in `ICDARBornDigitalTextDataset`
```
138,225,141,225,141,225,138,225,###
```

`gt_img_315.txt` in `ICDARBornDigitalTextDataset`
```
90,225,91,225,91,225,90,225,###
```

I can't understand why these bounding box exist as ground truth :anger:

To ignore these strange bounding box,<strike> use `Ignore(difficult=True)`</strike>
use `Ignore(strange=True)`
